config/environments/test.rb

```diff
@@ -66,6 +66,9 @@
   # Tell Active Support which deprecation messages to disallow.
   config.active_support.disallowed_deprecation_warnings = []
 
+  # Change to reduce unecessary logging
+  config.log_level = ENV.fetch("RAILS_TEST_LOG_LEVEL", :debug)
```

Comment:
Why don't we use the same variable `RAILS_LOG_LEVEL`?

I noticed that we use `DEV_LOG_LEVEL` but that's not a Rails thing. Matt added that in 78e28c517b6b93d320508121f95226d742a21ec2. Since we can use our different `.env` files, I don't see a reason for different names.

And then there's the option to only use STDOUT without a log file. People use different techniques but it seems to be a good idea to have an option to turn log files off in dev:

* https://discuss.rubyonrails.org/t/suggestion-prevent-rails-from-writing-development-test-log-files-by-default/72234

As alternative to setting a different log-level, you could just redirect: `ln -s /dev/null log/test.log`. Or does Apple not have /dev/null?